packages/rpc-core/src/rpc-methods/__tests__/get-multiple-accounts-test.ts

```diff
@@ -0,0 +1,591 @@
+import { Base58EncodedAddress } from '@solana/addresses';
+import { createHttpTransport, createJsonRpc } from '@solana/rpc-transport';
+import type { SolanaJsonRpcErrorCode } from '@solana/rpc-transport/dist/types/json-rpc-errors';
+import type { Rpc } from '@solana/rpc-transport/dist/types/json-rpc-types';
+import fetchMock from 'jest-fetch-mock-fork';
+
+import { Commitment } from '../common';
+import { createSolanaRpcApi, SolanaRpcMethods } from '../index';
+
+describe('getMultipleAccounts', () => {
+    let rpc: Rpc<SolanaRpcMethods>;
+    beforeEach(() => {
+        fetchMock.resetMocks();
+        fetchMock.dontMock();
+        rpc = createJsonRpc<SolanaRpcMethods>({
+            api: createSolanaRpcApi(),
+            transport: createHttpTransport({ url: 'http://127.0.0.1:8899' }),
+        });
+    });
+
+    (['confirmed', 'finalized', 'processed'] as Commitment[]).forEach(commitment => {
+        describe(`when called with \`${commitment}\` commitment`, () => {
+            it('returns account info for multiple accounts', async () => {
+                expect.assertions(3);
+
+                const multipleAccounts = await rpc
+                    .getMultipleAccounts(
+                        [
+                            // See scripts/fixtures/GQE2yjns7SKKuMc89tveBDpzYHwXfeuB2PGAbGaPWc6G.json
+                            'GQE2yjns7SKKuMc89tveBDpzYHwXfeuB2PGAbGaPWc6G' as Base58EncodedAddress<'GQE2yjns7SKKuMc89tveBDpzYHwXfeuB2PGAbGaPWc6G'>,
+                            // See scripts/fixtures/4nTLDQiSTRHbngKZWPMfYnZdWTbKiNeuuPcX7yFUpSAc.json
+                            '4nTLDQiSTRHbngKZWPMfYnZdWTbKiNeuuPcX7yFUpSAc' as Base58EncodedAddress<'4nTLDQiSTRHbngKZWPMfYnZdWTbKiNeuuPcX7yFUpSAc'>,
+                        ],
+                        {
+                            commitment,
+                        }
+                    )
+                    .send();
+
+                expect(multipleAccounts.value).toHaveLength(2);
+                multipleAccounts.value.forEach(accountInfo => {
+                    expect(accountInfo).toMatchObject({
+                        data: expect.any(Array),
+                        executable: expect.any(Boolean),
+                        lamports: expect.any(BigInt),
+                        owner: expect.any(String),
+                        rentEpoch: expect.any(BigInt),
+                        space: expect.any(BigInt),
+                    });
```

Comment:
Do we want to verify the account's state considering we have it stored as a fixture? ie. validating the proper lamports, owner, data, etc.

Reply:
We could do but have generally been pretty lazy with this in the other tests. I think if we did this we'd just be testing the data flow from fixtures into RPC responses really, I'm not sure we'd really improve coverage of our own code. I haven't thought about it too much though so not too sure.

Reply:
I'll leave it up to you for this one! Since we have to comb through all of the tests at some point, prob not a big deal